frontends/precinct-scanner/src/screens/setup_scanner_internal_wiring_screen.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {
+  ScreenMainCenterChild,
+  CenteredLargeProse,
+} from '../components/layout';
+
+export function SetupScannerInternalWiringScreen(): JSX.Element {
+  return (
+    <ScreenMainCenterChild infoBar={false}>
+      <CenteredLargeProse>
+        <h1>Scanner Not Detected</h1>
+        <p>
+          There is an internal connection problem, please report to election
+          clerk.
```

Comment:
The grade school grammar nerd in me wants to add a period here instead of a comma to avoid a comma splice :smile:

```
There is an internal connection problem. Please report to an election clerk.
```

Reply:
Good call!